packages/create-cloudflare/templates/next/workers/templates/wrangler.jsonc

```diff
@@ -1,12 +1,19 @@
 {
   "name": "<app-name>",
   "main": ".open-next/worker.js",
-  "compatibility_date": "2024-09-26",
+  "compatibility_date": "2025-03-01",
   "compatibility_flags": [
     "nodejs_compat"
   ],
   "assets": {
     "binding": "ASSETS",
     "directory": ".open-next/assets"
-  }
+  },
+  // R2 incremental cache
+  "r2_buckets": [
+    {
+      "binding": "NEXT_INC_CACHE_R2_BUCKET",
+      "bucket_name": "<BUCKET_NAME>"
+    }
+  ]
```

Comment:
So this will mean that out of the box this template cannot work, unless the developer provisions an R2 bucket, right?

Perhaps we should comment this out initially?

Reply:
You are absolutely, right, I'll do that

Reply:
Updated, thanks for catching!